pytest_operator/plugin.py

```diff
@@ -63,6 +63,19 @@ def pytest_addoption(parser):
         help="Whether to run charmcraft in destructive mode "
         "(as opposed to doing builds in lxc containers)",
     )
+    parser.addoption(
+        "--crash-dump",
+        action="store_true",
+        default=True,
+        help="Whether to run juju-crashdump after failed tests. "
+        "This is enabled by default.",
+    )
+    parser.addoption(
+        "--crash-dump-output",
+        action="store",
+        help="Store the completed crash dump in this dir. "
+        "The default is current folder.",
+    )
```

Comment:
Should we be explicit that the default is `None` which results in the crashdump being run in `cwd`?
```suggestion
    parser.addoption(
        "--crash-dump-output",
        action="store",
        default=None
        help="Store the completed crash dump in this dir. "
        "The default is current folder.",
    )
```